front/src/scenes/player.scene.tsx

```diff
@@ -1,6 +1,14 @@
 import * as React from 'react';
 import { PlayerContainer } from 'pods/player';
+import { CenteredLayout } from 'layout';
+import { Header, Footer } from 'common-app/components';
 
 export const PlayerScene = () => {
-  return <PlayerContainer />;
+  return (
+    <CenteredLayout>
+      <Header />
+      <PlayerContainer />
+      <Footer />
+    </CenteredLayout>
```

Comment:
Same here, we should reuse the AppLayout that you are going to create

Reply:
resoved in new commit